src/software/world/robot_state.h

```diff
@@ -82,6 +126,9 @@ class RobotState
     Vector velocity_;
     Angle orientation_;
     AngularVelocity angular_velocity_;
+    bool ball_in_beam_;
+    std::optional<unsigned long> time_since_last_chip_ms_;
+    std::optional<unsigned long> time_since_last_kick_ms_;
```

Comment:
Sorry, should've asked about this on my first pass, but why not use `Duration`?

Reply:
good idea, changed that